src/styles/default-settings.scss

```diff
@@ -0,0 +1,6 @@
+* {
+  margin: 0;
+  padding: 0;
+
```

Comment:
```suggestion

```
redundant to reset margin and padding for all selectors. instead better to reset only for elements where it is necessary